test/integration/golden_response/get_stat_ranking/country.json

```diff
@@ -20,1216 +20,1246 @@
           },
           {
             "rank": 4,
+            "value": 293047571,
+            "placeDcid": "country/SUN"
```

Comment:
this is also a regression - we should not include USSR in country rankings @Spaceenter

Reply:
Probably since cl/377096245 migrated place-ranking computation to SVObs version.  I think as we migrate pipelines to SVObs we should probably do eager manual validation of diffs (mixer diffs, and run cache-differ) before release.

Reply:
cl/379660649 should fix the bug.